apps/storybook/stories/Namekit/CurrencySymbol.stories.tsx

```diff
@@ -0,0 +1,117 @@
+import { Currency } from "@namehash/ens-utils";
+import { PriceDisplaySize, CurrencySymbol } from "@namehash/namekit-react";
+import type { Meta, StoryObj } from "@storybook/react";
+
+export const ETH: Story = {
+  args: {
+    currency: Currency.Eth,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const USD: Story = {
+  args: {
+    currency: Currency.Usd,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const USDC: Story = {
+  args: {
+    currency: Currency.Usdc,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const WETH: Story = {
+  args: {
+    currency: Currency.Weth,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const DAI: Story = {
+  args: {
+    currency: Currency.Dai,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const MicroSize: Story = {
+  args: {
+    size: PriceDisplaySize.Micro,
+    currency: Currency.Eth,
+  },
+};
+export const SmallSize: Story = {
+  args: {
+    size: PriceDisplaySize.Small,
+    currency: Currency.Eth,
+  },
+};
+export const MediumSize: Story = {
+  args: {
+    size: PriceDisplaySize.Medium,
+    currency: Currency.Eth,
+  },
+};
+export const LargeSize: Story = {
+  args: {
+    size: PriceDisplaySize.Large,
+    currency: Currency.Eth,
+  },
+};
+export const WithCustomSymbolColor: Story = {
+  args: {
+    symbolFillColor: "#007bff",
+    size: PriceDisplaySize.Large,
+    currency: Currency.Eth,
+  },
+};
+export const ShowingTooltipDescription: Story = {
```

Comment:
Why does this story cause the symbol to become horizontally aligned center?

Reply:
It is a layout configuration we can decide wether to put the Ui comps :)

Do you like it this way?

Reply:
@FrancoAguzzi I think it's better to have everything with a default (left) alignment in all storybook stories.